### PR TITLE
add network interface exclusions

### PR DESCRIPTION
**Pull request by nojnhuh**
Part of https://github.com/openservicemesh/osm/issues/4546

### CODE REVIEW

File: content/docs/guides/traffic_management/iptables_redirection.md [hunk 1]
```diff
@@ -196,3 +196,25 @@ kubectl annotate pod <pod> openservicemesh.io/inbound-port-exclusion-list=6379,7
...
+    osm install --set="osm.networkInterfaceExclusionList={eth0,net1}
+    ```
+
+1. By setting the `networkInterfaceExclusionList` field in the `osm-mesh-config` resource:
+    ```bash
+    ## Assumes OSM is installed in the osm-system namespace
+    kubectl patch meshconfig osm-mesh-config -n osm-system -p '{"spec":{"traffic":{"networkInterfaceExclusionList":["eth0", "net1"]}}}'  --type=merge
+    ```
+
+   When network interfaces are set for exclusion post-install, make sure to restart the pods in monitored namespaces for this change to take effect.
```
Comment (trstringer) [inline content/docs/guides/traffic_management/iptables_redirection.md:218]: Did you mean to have this indented?

File: content/docs/guides/troubleshooting/traffic_management/iptables_redirection.md [hunk 2]
```diff
@@ -180,3 +180,96 @@ Init Containers:
 In the example above, the following `iptables` commands are responsible for explicitly ignoring the configured outbound ports (`6379, 7070 and 8080`) from being redirected to the Envoy proxy sidecar.
 ```console
 iptables -t nat -I PROXY_OUTPUT -p tcp --match multiport --dports 6379,7070,8080 -j RETURN
+```
+
+## When network interface exclusions are configured
+
+By default, all traffic using TCP as the underlying transport protocol are redirected via the Envoy proxy sidecar container. This means all TCP based outbound traffic from applications are redirected and routed via the Envoy proxy sidecar based on service mesh policies. When network interface exclusions are configured, traffic belonging to these network interfaces will not be proxied to the Envoy sidecar.
+
+If network interfaces are configured to be excluded but being subject to service mesh policies, verify they are configured as expected.
```
Comment (trstringer) [inline content/docs/guides/troubleshooting/traffic_management/iptables_redirection.md:189]: This wording is a little confusing. Perhaps reword to something like "If you have network interfaces that are subjected to mesh policies, verify their configuration to ensure they are properly excluded." Or something like that.

**Review by trstringer — CHANGES_REQUESTED** [review]

File: content/docs/guides/troubleshooting/traffic_management/iptables_redirection.md (hunk 2 above)
Comment (nojnhuh) [reply content/docs/guides/troubleshooting/traffic_management/iptables_redirection.md:189]: Fixed

File: content/docs/guides/traffic_management/iptables_redirection.md (hunk 1 above)
Comment (nojnhuh) [reply content/docs/guides/traffic_management/iptables_redirection.md:218]: Yes, at least to match the `### 1. Global inbound port exclusions` section above.

**Review by shashankram — APPROVED** [review]

**Pull request merged**